JSCO4180/exercise_jasco4180.py

```diff
@@ -0,0 +1,27 @@
+# Exercise the JASCO PU-4180
```

Comment:
It would be useful to link to a wiki page explaning the issues in more detail here - it's not totally obvious what an "exercise" might be otherwise.

Reply:
I updated the file to include a more descriptive header. I also updated the wiki.